### ISSUE TITLE
As a user who uploaded a custom avatar, I do not want to get fooled about using Gravatar

### ISSUE THREAD

**Issue by franz-josef-kaiser** [wecodemore/wcm-avatar#54]
With WP 4.4, a description below a users avatar on its profile page was introduced. It is advertising for _Gravatar_ and of no use if the user already uploaded a custom avatar. As we already properly display a custom avatar there, we should change this.

<img width="569" alt="bildschirmfoto 2016-11-16 um 20 36 01" src="https://cloud.githubusercontent.com/assets/387666/20362559/5474f35e-ac3c-11e6-897f-4b6b68e8c822.png">

It only shows when `IS_PROFILE_PAGE` is set and `true`. The `user_profile_picture_description`-filter allows changing this string. We should do that:

```php
add_filter( 'user_profile_picture_description', function( $descr ) {

    if ( 
        defined( 'IS_PROFILE_PAGE' ) and IS_PROFILE_PAGE
        && apply_filters( 'wcm.avatar.enable_custom_avatar', TRUE )
    ) {
        $has = get_user_meta( 
            get_current_user_id(), 
            apply_filters( 'wcm.avatar.meta_key', 'user_avatar' ),
            TRUE 
        );
        $descr = $has
            ? 'You can change your avatar by defining a new one <a href="#tmpl-main--container">here</a>.'
            : 'You can upload an avatar <a href="#tmpl-main--container">on top</a>.';
    }

    return $descr;
} );
```

/Cc @stephenharris Maybe you got some input on how to phrase the description?

**Comment by kraftner** [comment]
Just some thoughts:

Do we really need the `IS_PROFILE_PAGE` check? It looks like this filter isn't used anywhere else anyway.

Concerning the wording I'd make it consistent.

```
'You can change your avatar by uploading a new one <a href="#tmpl-main--container">here</a>.'
'You can upload an avatar <a href="#tmpl-main--container">here</a>.';
```

Also I think we should avoid mentioning any location considering #31

**Comment by franz-josef-kaiser** [comment]
@kraftner Yes, the check is needed to use the correct function to retrieve the ID of the right user – something that is missing in above example.

**Comment by franz-josef-kaiser** [comment]
About the localisation: Did you link the wrong ticket?

**Comment by kraftner** [comment]
Ahh, now I get it. Thanks for the clarification.

> About the localisation: Did you link the wrong ticket?

No I didn't and I said location, not localisation. :smile: 
If we make the positioning of the UI flexible as proposed in #31 using the word "top" could be wrong, so better avoid any wording like that.

**Comment by franz-josef-kaiser** [comment]
Ah, got it! :D

**Comment by kraftner** [comment]
I think this issue would be ready for a PR, @franz-josef-kaiser ?
